src/components/calcite-action/calcite-action.scss

```diff
@@ -50,6 +50,15 @@
   background-color: var(--calcite-app-background-active);
 }
 
+:host([disabled]) .button,
+:host([disabled]) .button:hover {
+  cursor: default;
+  background-color: gray;
+  fill: white;
+  color: white;
+  /* todo for Alan */
```

Comment:
Use issue/PR instead of comment?

Reply:
https://github.com/Esri/calcite-app-components/issues/170